src/components/protection/AddProtectionSingle.vue

```diff
@@ -365,9 +364,26 @@ export default class AddProtectionSingle extends BaseComponent {
         onUpdate: this.onUpdate
       });
       this.success = txRes;
+      addNotification(
+        "Add Single-Sided Liquidity",
+        `Add ${this.prettifyNumber(this.amount)} ${this.token.symbol} to ${
+          this.pool.name
+        } pool,`,
+        undefined,
```

Comment:
Yeah screw that, lets definitely make this ```addNotification``` function accept a single object instead.